sdk/test/Services/S3/UnitTests/Custom/MultipartDownloadManagerTests.cs

```diff
@@ -1951,74 +1951,6 @@ public async Task DiscoverUsingRangeStrategy_CallsWaitForCapacityAsync()
             mockDataHandler.Verify(x => x.WaitForCapacityAsync(It.IsAny<CancellationToken>()), Times.Once);
         }
 
-        [TestMethod]
-        public async Task DiscoverUsingPartStrategy_AcquiresAndReleasesHttpSlot()
```

Comment:
these tests are not applicable because they were asserting the old behavior of releasing the http slot only after we get the initial response, and not after processing